cli/test/lib/commands/publish.test.ts

```diff
@@ -0,0 +1,150 @@
+
+import env from '@cube-creator/core/env'
+import { before, describe, it } from 'mocha'
+import { expect } from 'chai'
+import $rdf from 'rdf-ext'
+import { CONSTRUCT } from '@tpluscode/sparql-builder'
+import debug from 'debug'
+import { dcat, dcterms, rdf, schema, sh, vcard, xsd } from '@tpluscode/rdf-ns-builders'
+import { setupEnv } from '../../support/env'
+import { client, insertTestData } from '@cube-creator/testing/lib'
+import { cube, scale } from '@cube-creator/core/namespace'
+import clownface from 'clownface'
+import publish from '../../../lib/commands/publish'
+import namespace from '@rdfjs/namespace'
+
+describe('lib/commands/publish', function () {
+  this.timeout(200000)
+
+  const ns = {
+    targetCube: namespace('https://environment.ld.admin.ch/foen/ubd/28/'),
+  }
+
+  const executionUrl = 'http://example.com/transformation-test'
+
+  before(async () => {
+    setupEnv()
+    await insertTestData('fuseki/sample-ubd.trig')
+  })
+
+  it('produces triples', async function () {
+    // given
+    const runner = publish($rdf.namedNode('urn:pipeline:cube-creator#Main'), debug('test'))
+
+    // when
+    await runner({
+      debug: true,
+      job: `${env.API_CORE_BASE}cube-project/ubd/csv-mapping/jobs/test-publish-job`,
+      executionUrl,
+    })
+    await new Promise((resolve) =>
+      setTimeout(resolve, 100),
+    )
+
+    // then
+    const expectedGraph = ns.targetCube()
+    const dataset = await $rdf.dataset().import(await CONSTRUCT`?s ?p ?o`
+      .FROM(expectedGraph)
+      .WHERE`?s ?p ?o`
+      .execute(client.query))
+
+    const cubePointer = clownface({ dataset })
```

Comment:
Please wrap the test setup code in an additional `describe+before` and make put the expectations in their own `it` blocks

```suggestion
  describe('produces triples', async function () {
    // given
    const runner = publish($rdf.namedNode('urn:pipeline:cube-creator#Main'), debug('test'))

    // when
    await runner({
      debug: true,
      job: `${env.API_CORE_BASE}cube-project/ubd/csv-mapping/jobs/test-publish-job`,
      executionUrl,
    })
    await new Promise((resolve) =>
      setTimeout(resolve, 100),
    )

    // then
    const expectedGraph = ns.targetCube()
    const dataset = await $rdf.dataset().import(await CONSTRUCT`?s ?p ?o`
      .FROM(expectedGraph)
      .WHERE`?s ?p ?o`
      .execute(client.query))

    const cubePointer = clownface({ dataset })
```